src/pybind/mgr/volumes/module.py

```diff
@@ -46,8 +46,10 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
         {
             'cmd': 'fs subvolumegroup create '
                    'name=vol_name,type=CephString '
-                   'name=group_name,type=CephString ',
-            'desc': "Create a CephFS subvolume group in a volume",
+                   'name=group_name,type=CephString '
+                   'name=mode,type=CephString,req=false ',
```

Comment:
@batrick, do you want this to be of type 'CephInt' ?

Reply:
I think we have to accept a string because there's no way to tell the command parser to treat the argument as octal. It probably is smart enough to handle "0777" correctly but then the user needs to remember to add the "0" which is not required for other tools like chmod.

Reply:
We can also change this later. Not worth overthinking this when we need to get this merged so we can backport to 14.2.2.